src/main/resources/swagger/api-docs.yaml

```diff
@@ -222,6 +222,164 @@ paths:
       tags:
         - Group
 
+  /api/group/{groupName}/{policyName}:
+    get:
+      summary: 'Get email addresses for members of the "admin" policy'
+      responses:
+        200:
+          description: 'Email addresses in the "admin" policy'
+          schema:
+            $ref: '#/definitions/ArrayOfEmails'
+        404:
+          description: Group could not be found or you do not have the required permissions on this group
+        500:
+          description: Internal Error
+          schema:
+            $ref: '#/definitions/ErrorReport'
+      parameters:
+        - in: path
+          description: Name of group
+          name: groupName
+          required: true
+          type: string
+        - in: path
+          description: Name of policy
+          name: policyName
+          required: true
+          type: string
+          enum: ["members", "admins"]
+      operationId: GetGroupAdminEmails
+      tags:
+        - Group
+    put:
+      summary: 'Overwrite email addresses of members of the "admin" policy'
+      responses:
+        201:
+          description: Admin Policy successfully updated
+        404:
+          description: Group does not exist or you are not a member of the admin policy for the group
```

Comment:
a leftover from merging the two endpoints into one: a bunch of these descriptions specifically reference the "admin" policy

Reply:
fixed